services/forms-api/lambdas/getForm.js

```diff
@@ -23,14 +24,14 @@ export async function main(event) {
 
   const formObj = formQueryToObj(queryResponse.Items);
   const { id, children: relationships, ...formAttributes } = formObj[formPartitionKey];
-  return response.success(200, {
-    type: 'forms',
+  const formdata = {
     id,
     attributes: {
       ...formAttributes,
     },
```

Comment:
We could just pass down attributes here instead of formAttributes, so just rename ...formAttributes to ...attributes :D